Missing traits for Sorgenfrey line (#1258)
pi-base/data: spaces/S000043/properties/P000206.md

---
space: S000043
property: P000206
value: true
---

Player 2 has the following winning strategy:

For every $n<\omega$ it is enough to pick $V_n:=[x_n,y_n)$, where $[x_n,y_n]\subset U_n$, $y_n>x_n$.

> Then $\bigcup_{n=0}^\infty U_n \supset \bigcap_{n=0}^\infty [x_n,y_n]$ is not empty

Reviewer: ```suggestion
Then $\bigcap_{n=0}^\infty U_n \supset \bigcap_{n=0}^\infty [x_n,y_n]$ is not empty
```

> Then $\bigcap_{n=0}^\infty U_n \supset \bigcap_{n=0}^\infty [x_n,y_n]$ is not empty
> as the intersection of compact sets in the Euclidean topology.

Reviewer 2: ```suggestion
Then $\bigcap_{n=0}^\infty U_n \supset \bigcap_{n=0}^\infty [x_n,y_n]$, which is nonempty
as a nested intersection of nonempty compact sets in the Euclidean topology.
```
